docs/proposals/config_data.md

```diff
@@ -173,7 +173,36 @@ type ConfigDataSelector struct {
 
 ### Volume Source
 
-The volume source will be addressed in a follow-up PR.
+A new `ConfigDataVolumeSource` type of volume source containing the `ConfigData` object will be
+added to the `VolumeSource` struct in the API:
+
+```go
+package api
+
+type VolumeSource struct {
+  // other fields omitted
+  ConfigData *ConfigDataVolumeSource `json:"configData,omitempty"`
+}
+
+// ConfigDataVolumeSource represents a volume that holds configuration data
+type ConfigDataVolumeSource struct {
+  // A list of config data keys to project into the volume in files
+  Files []ConfigDataVolumeFile `json:"files"`
```

Comment:
DownwardAPI says "Items".  It probably should have said "Files", but it doesn't.  Consistency is more important than correctness.  File a bug to fix this for v2 api (seriously :)